src/common/Makefile.am

```diff
@@ -103,10 +99,8 @@ libcommon_internal_la_SOURCES += \
 endif
 
 # used by RBD and FileStore
-if LINUX
```

Comment:
@trociny, no FileStore is not built for client side as I understand. Due to limitation on Makefile.am syntax, probably best to add another if FREE_BSD or something. Not sure why WITH_RBD guard was taken away.